src/main/java/org/folio/linked/data/integration/kafka/sender/search/AuthorityCreateMessageSender.java

```diff
@@ -35,18 +38,33 @@ public class AuthorityCreateMessageSender implements CreateMessageSender {
 
   @Override
   public Collection<Resource> apply(Resource resource) {
-    return Stream.of(resource)
-      .filter(this::test)
+    if (isAuthority(resource)) {
```

Comment:
```
isNotIndexedAuthority(Resource resource) {
           return singletonList(resource);
}
return findNotIndexedAuthoritiesInResource(resource);
```

Reply:
we want to return empty list in case of indexDate != null